scripts/generator/graphql_generator/csharp/SDK/DefaultProductQueries.cs.erb

```diff
@@ -7,7 +7,7 @@ namespace Shopify.Unity.SDK {
     /// Generates default queries for <see ref="ShopifyClient.products">ShopifyClient.products </see>.
     /// </summary>
     public class DefaultProductQueries {
-        public void ShopProducts(QueryRootQuery query, Dictionary<string,int> imageResolutions, int first = 250, string after = null) {
+        public void ShopProducts(QueryRootQuery query, Dictionary<string,int> imageResolutions, int first = 20, string after = null) {
```

Comment:
We define `20` as a constant here https://github.com/Shopify/unity-buy-sdk/pull/387/files#diff-4056dfc8033dc677ee07c55e6dfe0bc6R11 which is 👍 but then here we use it as a magic number 👎 Can we reference the constant in all places? (Then if we change it, we only need to make the update in one place, therefore preventing these getting out of sync.)

Reply:
It's unfortunate but in C# there's now way to do `#define PAGE_SIZE 20` and you can't have a variable as the default for optional types.